tests/Test/AbstractIntegrationTestCase.php

```diff
@@ -51,13 +51,16 @@
  * {"indent": "    ", "lineEnding": "\n"}
  * --SETTINGS--*
  * {"key": "value"} # optional extension point for custom IntegrationTestCase class
+ * --REQUIREMENTS--*
+ * {"php": 70400**, "php<": 80000***}
  * --EXPECT--
  * Expected code after fixing
  * --INPUT--*
  * Code to fix
  *
- *   * Section or any line in it may be omitted.
- *  ** PHP minimum version. Default to current running php version (no effect).
+ *   *  Section or any line in it may be omitted.
+ *  **  PHP minimum version. Default to current running php version (no effect).
+ *  *** PHP upper limit, that is first too-high version. Default is empty (no effect).
```

Comment:
Using asterisk here is IMHO totally unreadable, because whole phpDoc is behind `*`. Single asterisk after section marker can't be there in actual test case, also `**` and `***` makes JSON invalid. I would totally change it since you're cleaning this up.

```suggestion
 * IMPORTANT:
 *   - Some sections (like `--CONFIG--`) may be omitted. The required sections are: `--TEST--`, `--RULESET--` and `--EXPECT--` (which works as input too if `--INPUT--` is not provided, that means no changes are expected).
 *   - `--REQUIREMENTS--` section can define additional constraints for running (or not) the test. You can use these fields to fine-tune run conditions for test cases:
 *     - `php` represents minimum PHP version test should be run on. Default to current running PHP version (no effect).
 *     - `php<` represents maximum PHP version test should be run on. Default to PHP's maximum integer value (no effect).
```

The asterisks must be removed from the snippet, but I can't make full suggestion here.

Reply:
I'm not in favour of manually listing which section is obligatory instead marking it in format itself.
if the full suggestion is not working here for you, maybe raise a dedicated PR?

Reply:
But it makes format malformed and unclear, since asterisks are either on the left (phpDoc) and on the right ("annotations"). Your choice 🤷‍♂️.

Reply:
I extended the already existing list. If you want to change syntax of this list, please raise a new PR. for me, out of scope of this PR, following your reasoning from previous PRs :D

Reply:
Fair enough 😉.